src/analyzer/protocol/conn-size/events.bif

```diff
@@ -23,5 +24,25 @@ event conn_bytes_threshold_crossed%(c: connection, threshold: count, is_orig: bo
 ## is_orig: true if the threshold was crossed by the originator of the connection
 ##
 ## .. zeek:see:: set_current_conn_packets_threshold set_current_conn_bytes_threshold conn_bytes_threshold_crossed
-##              get_current_conn_bytes_threshold get_current_conn_packets_threshold
+##               get_current_conn_bytes_threshold get_current_conn_packets_threshold conn_duration_threshold_crossed
+##               set_current_conn_duration_threshold get_current_conn_duration_threshold
 event conn_packets_threshold_crossed%(c: connection, threshold: count, is_orig: bool%);
+
+## Generated for a connection that crossed a set duration threshold. Note that this
+## is a low level event that should usually be avoided for user code. Use
+## ConnThreshold::duration_threshold_crossed instead.
```

Comment:
Nice to use a `:zeek:see:` so an HTML link will actually be generated for "ConnThreshold::duration_threshold_crossed"